README.md

```diff
@@ -18,6 +18,7 @@ If you have never contributed to GitHub before, [here is a guide](http://ablwr.g
 - [Collective Access Community: Guides and Workflows](http://collectiveaccesscommunity.org/category/guides/)
 - [Digital Processing, Bentley Historical Library](https://sites.google.com/a/umich.edu/bhl-archival-curation/processing-archival-collections/08-digital-processing): Workflow for ingesting digital objects at the Bentley Historial Library.
 - [Format Migrations at Harvard](http://blogs.loc.gov/thesignal/2015/04/format-migrations-at-harvard-library-an-ndsr-project-update/)
+- [Indigitization](http://www.indigitization.ca/indigitization-toolkit/introduction/) A 'toolkit' of workflows focussed on supporting First Nations communities' digitization of cultural resources.  Contains detailed walkthroughs for Cassette digitization as well as metadata best practices.
```

Comment:
I suppose my personal taste in typographic style must bow to the consistency of the document!  I'll edit that out!